addons/mail/static/src/models/channel.js

```diff
@@ -32,6 +32,9 @@ registerModel({
          * @returns {FieldCommand}
          */
         _computeCallParticipants() {
+            if (!this.thread) {
+                return;
```

Comment:
`return clear();` ?

Reply:
It's just a temporary state to avoid crash, we never intend to remove the thread and hope for the participant to become empty. But I'll add it.